### PR TITLE
 PCG-11-styling-appBar-page 

### PR DESCRIPTION
**Pull request by KolyaIvankov22**
install @mui/icons-material 

edit native colors from MUI to our custom colors from layout

add stylling for appBar page with using MUI

make appBar responsive and add burger icon menu

### CODE REVIEW

File: src/components/appBar/AppBar.tsx [hunk 1]
```diff
@@ -1,34 +1,109 @@
...
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 500);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
```
Comment (makandriival) [inline src/components/appBar/AppBar.tsx:32]: i like this solution. Smart.
at the same time you probably will use the same logic in other components to.
In my experience we usually put it in a custom hook, if you need tip on how to do it, lmk ;) 
at the same time of you wanna leave it as it is, its okay by me, we can fix it later

File: src/components/appBar/AppBar.tsx [hunk 2]
```diff
@@ -1,34 +1,109 @@
...
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+          }}
+        >
+          <Box>
+            <img src={logo} alt="Logo" />
+          </Box>
+          {!isMobile && (
```
Comment (makandriival) [inline src/components/appBar/AppBar.tsx:58]: 1. the condition you did here is doable with one ternary, which in my humble opinion is a bit more neat
2. i would create separate components for mobile and desktop view's, just to split the code and have nice and readable little components

**Review by makandriival — APPROVED** [review]
good job with mui 😊
i left couple comments, no critical mistakes,
but there is room for improvement.
also, appbar definitely needs more styling, margins are to little when it switches to burger btn

i will approve because the code is working properly,
if you not in rush pls fix my comments ;)

**Review by VitiukVV — APPROVED** [review]

File: src/components/appBar/AppBar.tsx (hunk 1 above)
Comment (KolyaIvankov22) [reply src/components/appBar/AppBar.tsx:32]: Yeahh, it will be a great to see how it should look like)

**Pull request merged**